src/OpenTelemetry.Exporter.OpenTelemetryProtocol/.publicApi/netstandard2.1/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,4 @@
+OpenTelemetry.Exporter.OtlpLogExporter
```

Comment:
there is public API change in the core OTLP package, right? We'll need to grant internalsvisible to the OTLP.Logs project

Reply:
I got the public API files fixed up. No public API changes to the core package. InternalsVisibleTo has already been applied.